maps/tradeship/tradeship.dm

```diff
@@ -8,6 +8,9 @@
 	#define CULTURE_SCAV_ENCLAVE     "Enclave Yinglet"
 	#define CULTURE_SCAV_TRIBE       "Tribal Yinglet"
 	#define FACTION_SCAV             "Scav"
+	#define LANGUAGE_BAXXID          "Baxxid"
+	#define CULTURE_BAXXID           "Baxxid"
```

Comment:
Keys need to be unique or they will overwrite each other in the index list. 'Baxxid Clans' or something would be sufficient.

Reply:
Changed definition of CULTURE_BAXXID from "Baxxid" to "Baxxid Clans" in tradeship.dm (line 12)